pkg/controllers/user/helm/controller.go

```diff
@@ -305,6 +299,12 @@ func (l *Lifecycle) Run(obj *v3.App, template, templateDir, notes string) error
 		return err
 	}
 	if err := helmInstall(templateDir, kubeConfigPath, obj); err != nil {
+		// create an app revision so that user can decide to continue
+		err2 := l.createAppRevision(obj, template, notes, true)
```

Comment:
Not sure if there is a nicer way to handle this?

Reply:
should be enough. Just want to make sure we handle this error

Reply:
can you expand on your comment and explain why the app revision is important here